jmclient/jmclient/maker.py

```diff
@@ -605,11 +605,7 @@ def on_tx_received(self, nick, txhex):
                      "value": new_change_amount})
         new_ins = [x[1] for x in utxo.values()]
         new_ins.extend(my_utxos.keys())
-        # set locktime for best anonset (Core, Electrum) - most recent block.
-        # this call should never fail so no catch here.
-        currentblock = jm_single().bc_interface.rpc(
```

Comment:
As @chris-belcher pointed out in another recent PR thread, it's a failure of the abstraction to be referring to an rpc method here, it should be the business of the blockchaininterface (not even the walletservice, really - although currently there are rpc calls there, we should change that albeit it's not urgent) that rpc is being used, client shouldn't know about it. Could you maybe create a get_block_height() or similar function in walletservice which the get_default_locktime() uses, and is also called here?